Jenkinsfile

```diff
@@ -680,19 +615,6 @@ pipeline {
                         buildHipClangJobAndReboot(build_type: 'debug', config_targets: Smoke_targets)
                     }
                 }
-                stage('Fp32 OpenCL Debug gfx1030') {
```

Comment:
We do not have FP32 gfx1030 Smoke test anymore. This must be converted to HIP BE instead of removing.